### DISCUSSION TITLE
Q: netlify无法设置SITE_NAME环境变量 || Q: netlify cannot set the SITE_NAME environment variable

### DISCUSSION THREAD

**Discussion by wzdc** [walinejs/waline#2098] [Q&A] [answered]
### Describe the bug | 问题描述

Q: netlify无法设置SITE_NAME环境变量

### Website URL | 问题网站

.

### Where your waline deploy? | 服务部署在哪里？

Netlify

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

Q: netlify cannot set the SITE_NAME environment variable

### Website URL | Problem website

.

### Where your waline deploy? | Where is the service deployed?

Netlify

### Where your comment data store? | Where is the data stored?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
是的，SITE_NAME 是 Netlify 的官方变量，可以直接修改 Netlify 项目名称

---
Yes, SITE_NAME is an official variable of Netlify and you can directly modify the Netlify project name

**Reply by wzdc** [reply]
项目名称不支持中文而且会跟默认域名一同被修改

---
The project name does not support Chinese and will be modified together with the default domain name

**Reply by lizheming** [reply]
那就用英文吧

---
Then use English

**Reply by wzdc** [reply]
改了项目名称还是不生效，名称还是原来的

---
Changing the project name still does not take effect, the name is still the original

**Reply by lizheming** [reply]
重新部署

---
Redeploy
